### DISCUSSION TITLE
《第二十条》影评

### DISCUSSION THREAD

**Discussion by giscus** [ZexWoo/Giscus#31] [Announcements]
# 《第二十条》影评

给法律新风向吹风的作品。内容可能涉及剧透，如果你计划自己观看，就请不要阅读我的观后感。

https://zexwoo.blog/posts/gossip/article-20-review/

**Comment by 120318** [comment]
我觉得，最出戏的部分就是赵丽颖的白腿。

**Reply by 120318** [reply]
但是她演的还不错。
